tests/src/ClientTest.php

```diff
@@ -33,4 +34,24 @@ public function testContainer(): void
         $this->assertInstanceOf(TokenEndpoint::class, $container->get('auth'));
         $this->assertInstanceOf(TokenAwareInterface::class, $container->get('main'));
     }
+
+    /**
+     * @covers ::setToken
+     * @covers ::getToken
+     */
+    public function testToken(): void
+    {
+        $client = $this->getTestingClient([
+            'mainApiEndpoint' => 'https://example.com/v2/CheckConnection',
```

Comment:
Perhaps we should throw this dummy config into some trait constant, of default argument? At least for the cases when it's completely irrelevant

Reply:
I like the idea, but we have these all over, ex 'mainApiEndpoint' we can find also in data providers, so if we add them we can't do them in bulk, we need for each config a separate key
, ex `$config = ['mainApiEndpoint'=>'https://example.com/v2/CheckConnection']`
trait constant are avail only starting php8.2

Reply:
True, maybe let's wait for the rest of the endpoints, we'll see how it looks then